data/free worlds start.txt

```diff
@@ -1399,6 +1399,119 @@ mission "FW Pirates 3.1B"
+			decline
+			`	(Liberate <planet>'s slaves, wipe out the cartel palaces, and destroy the pirates' infrastructure.)`
+				`	Hovering over the pirate spaceport, you demand that the pirates immediately free their slaves, abandon their ships on the ground, and evacuate their spaceport. At first, they seem to comply. Then you get a frantic radio call from a surviving rebel ship. A massive pirate armada just jumped in, and another is launching from hidden hangers all over the planet!  A gravelly voice chuckles over the radio. "If you want our ships, come and get them!"`
+				`	Alarms scream. You look at your tactical display and see a massive salvo of missiles coming from the largest pirate fleet you've ever seen. It looks as if the pirate cartels have been holding their elite ships in reserve, and they've called in a lot of favors with some outside groups to protect their way of life. The local defense fleet you already defeated was just the vanguard for this armada. Half the pirates and marauders in the galactic south must be in the system`
+				`	It also looks as if they've brought some friends along. You notice a Republic carrier emitting strange energy signatures and launching fighters that are clearly alien in design.`
+				accept
+	on accept
+		set "fw pirates: own way"
+
+	npc kill
+		personality plunders staying
+		government "Pirate"
+		system Shaula
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		ship "Bastion (Heavy)" "Dread"
+		ship "Carrier (Alpha)" "Trafficker"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
```

Comment:
The alpha carrier is intended to be an alpha carrier. The way I figure, they need a place they can quietly traffic fertile women from in order to perpetuate themselves. When the slave market is threatened, the alphas have a vested interest in defending it. I've added some text that explains this at the end of the new Senate Hearing mission.

I understand that Marauder ships are meant to be special. This mission is intended to be a unique one. My lore reason for including them is to have tricked-out ships used by the senior people in the cartels, not the standard pirates. They're too valuable to risk normally, but when the player threatens to destroy their way of life, the cartels throw all the best of what they have into the fray.

My mission-design reason for including marauder ships is that I want the player to be challenged. Having stronger ships is a way to increase the difficulty without increasing the burden on the software/hardware of the computer running the game. The engine falconsin particular are intended to be a way of challenging a player who simply runs away to recharge shields. 

There may be a concern that this mission may provide the player with an opportunity to capture a lot of rare ships in one fell swoop. I don't think this is the case, though, since the player has only 1 day to fight this battle. Even a fully loaded Bactrian will run out of crew after capping a few large marauder ships. 

I see what you're saying about the ship count, though. I'll scale back the marauder fleets by half